ecl2df/bulk.py

```diff
@@ -0,0 +1,197 @@
+import sys
+import logging
+import argparse
+from pathlib import Path
+import importlib
+from inspect import signature, Parameter
+from typing import List
+from fmu.config.utilities import yaml_load
+from ecl2df.constants import SUBMODULES
+
+logger = logging.getLogger(__name__)
+
+
+standard_options = {
+    "initvectors": None,  # List[str]
+    "keywords": None,  # List[str] x3
+    "keyword": "VFPPROD",
+    "vfpnumbers": "",
+    "fipname": "FIPNUM",  # str
+    "vectors": "*",  # List[str]
+    "stackdates": False,  # bool x2
+    "dropconstants": False,  # bool
+    "arrow": False,  # bool x2
+    "coords": False,  # bool
+    "pillars": False,  # bool
+    "region": "",  # str
+    "rstdates": "",  # str
+    "soilcutoff": 0.5,  # float
+    "sgascutoff": 0.5,  # float
+    "swatcutoff": 0.5,  # float
+    "group": False,  # bool
+    "wellname": None,  # str
+    "date": None,  # str
+    "time_index": "raw",  # str
+    "column_keys": None,
+    "start_date": "",  # str
+    "startdate": None,
+    "end_date": "",  # str
+    "params": False,  # bool
+    "paramfile": None,  # str
+    "include_restart": False,  # bool
+    "boundaryfilter": False,
+    "onlyk": False,
+    "onlyij": False,
+    "nnc": False,
+    "verbose": False,
+    "zonemap": "tut",
+    "use_wellconnstatus": False,
+    "excl_well_startswith": None,
+}
+
+
+def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
+    """Set up sys.argv parsers.
+
+    Arguments:
+        parser (argparse.ArgumentParser or argparse.subparser): parser to
+            fill with arguments
+    """
+    # parser.add_argument("DATAFILE", help="Name of Eclipse DATA file.")
+    parser.add_argument("-v", "--verbose", action="store_true", help="Be verbose")
+    return parser
+
+
+def bulk_export(eclpath, config_path, include: List = None, options: dict = None):
+    """Bulk uploads every module to sumo with metadata
```

Comment:
where does sumo come into play in this function? Is this functionality only available with sumo?